Kinvey/Kinvey/DataStore.swift

```diff
@@ -229,8 +249,19 @@ open class DataStore<T: Persistable> where T: NSObject {
      - returns: A `Request` instance which will allow cancel the request later
      */
     @discardableResult
-    open func find(_ query: Query = Query(), deltaSet: Bool? = nil, readPolicy: ReadPolicy? = nil, completionHandler: @escaping ArrayCompletionHandler) -> Request {
-        return find(query, deltaSet: deltaSet, readPolicy: readPolicy) { (result: Result<[T], Swift.Error>) in
+    open func find(
+        _ query: Query = Query(),
+        deltaSet: Bool? = nil,
```

Comment:
agree, we don't need to decide which approach to do according to the previous comment.